pycape/experimental/cli.py

```diff
@@ -31,8 +31,8 @@ async def deploy(
             a Cape function declared in an app.py script.
         url: The Cape platform's websocket URL, which is responsible for forwarding
             client requests to the proper enclave instances. If None, tries to load
-            value from the ``CAPE_ENCLAVE_HOST`` environment variable. If no such
-            variable value is supplied, defaults to ``"wss://enclave.capeprivacy.com"``.
+            value from the ``ENCLAVE_HOST`` environment variable. If no such
```

Comment:
the environment variable is actually `CAPE_ENCLAVE_HOST`, as mentioned in [the comment in `pycape._config`
](https://github.com/capeprivacy/pycape/blob/7773179364e0ebc117e09f2215de6e6f3f23d1d8/pycape/_config.py#L6-L7)
```suggestion
            value from the ``CAPE_ENCLAVE_HOST`` environment variable. If no such
```

Reply:
ah good catch, I missed the prefix here: https://github.com/capeprivacy/pycape/blob/7773179364e0ebc117e09f2215de6e6f3f23d1d8/pycape/_config.py#L19